customisation/tokenisation.md

```diff
@@ -1,7 +1,26 @@
 ---
 grand_parent: Building and research
 parent: Customisation
-layout: coming_soon
 title: Tokenisation
-description:
+description: Splitting a string into a sequence of tokens
 ---
+
+Tokenisation is the process of splitting a string into a sequence of tokens.
+Tokens include words as well as punctuation characters.
```

Comment:
Do you think adding a sentence explaining what a `token` is would be useful here?
Something like...

```
Tokens are string elements that carry meaning.
Tokens include words as well as punctuation characters.
```

Reply:
Perhaps we could differentiate the meaning of `token` vs `word type`, so that readers know that `tokens` include repetitions.

In the [PR for word embeddings](https://github.com/machinetranslate/machinetranslate.org/pull/350), we mention both `vector` and `word embedding` concepts. Do you think we can link `word embedding` here, if we mention `word type`? Or perhaps that deserves a new article altogether.

Reply:
Oh, thanks I see it's problematic to introduce the concept of a token that way, let me try revising that.

I'll see if I can downplay the distinction between `token` and `word` or `word instance` because most people use the terms interchangeably

Reply:
Great, thanks!